lib/endpoint/find-parameters.js

```diff
@@ -227,3 +228,15 @@ function makeRelativeLinksAbsolute (state) {
   })
   return state
 }
+
+function replaceTimeNowDefault (state) {
+  state.results.forEach(output => {
+    for (let i = 0; i < output.params.length; i++) {
```

Comment:
I don’t think you need to iterate using a `for () {}` loop here, did you try to use `output.params.forEach()`?